test/providers/FakeRequestProvider.scala

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
We have already the providers in `test.utils.providers`

Reply:
This was done locally before the changes for the other API were merged to main. Removed